drmozzarella/urls.py

```diff
@@ -1,5 +1,8 @@
-from django.urls import path, include
+from django.urls    import path, include
+
+from products.views import MetaView
 
 urlpatterns = [
-    path('accounts', include('accounts.urls')),
-    ]
\ No newline at end of file
+    path('meta'     , MetaView.as_view()),
```

Comment:
제가 처음에 meta data니까 meta로 가는 것이 좋다고 알려 드렸었는데, 
주고 있는 데이터나, restful을 따져 봤을 때 다음과 같이 바꾸는 것이 좋을 것 같아요.
```suggestion
    path('menus'     , MetaView.as_view()),
```

Reply:
수정하였습니다.